addons/point_of_sale/static/src/js/models.js

```diff
@@ -414,6 +415,27 @@ exports.PosModel = Backbone.Model.extend({
 
         },
     },  {
+        model:  'pos.payment.method',
+        fields: ['name', 'is_cash_count', 'cash_journal_id', 'receivable_account_id'],
+        domain: function(self, tmp) {
+            return [['id', 'in', tmp.payment_method_ids]];
+        },
+        loaded: function(self, payment_methods) {
+            self.payment_methods = payment_methods.sort(function(a,b){
+                // prefer cash payment_method to be first in the list
+                if (a.is_cash_count && !b.is_cash_count) {
+                    return -1;
+                } else if (!a.is_cash_count && b.is_cash_count) {
+                    return 1;
+                } else {
+                    return a.id - b.id;
+                }
+            });
+            self.payment_methods_by_id = Object.fromEntries(payment_methods.map(
```

Comment:
don't really understand why you are needing 'payment_methods_by_id', if it is just to find the payment_method by id, we can probably just use _.find where it is needed

Reply:
In screen.js, payment_method with corresponding id performed several times everytime the user selects a payment method. I think it is better to have a mapping of the id to the payment method than always executing a _.find in the payment_methods.